cli/command/trust/sign_test.go

```diff
@@ -282,3 +282,19 @@ func TestPrintOtherSigners(t *testing.T) {
 	out := <-outC
 	assert.EqualValues(t, "Other signers of this tag:\nAlice, Bob, Carol\n", out)
 }
+
+func TestChangeList(t *testing.T) {
+
+	tmpDir, err := ioutil.TempDir("", "docker-sign-test-")
+	assert.NoError(t, err)
+	defer os.RemoveAll(tmpDir)
+	config.SetDir(tmpDir)
+	cmd := newSignCommand(
+		test.NewFakeCli(&fakeClient{}))
+	cmd.SetArgs([]string{"ubuntu:latest"})
+	cmd.SetOutput(ioutil.Discard)
```

Comment:
the command isn't executed in the test, I think you need to add a call to `cmd.Exec`